SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorLayoutDetails.cpp

```diff
@@ -51,7 +81,7 @@ FReply FSpatialGDKEditorLayoutDetails::ClickedOnButton()
 	{
 		UE_LOG(LogSpatialGDKEditorLayoutDetails, Warning, TEXT("Unable to generate a development authentication token. Result: %s"), *CreateDevAuthTokenResult);
 		FMessageDialog::Open(EAppMsgType::Ok, FText::FromString(FString::Printf(TEXT("Unable to generate a development authentication token. Result: %s"), *CreateDevAuthTokenResult)));
-		return FReply::Handled();
+		return FReply::Unhandled();
```

Comment:
What was the reason for this? Presumably nothing else is going to handle this, or is this considered best practice for slate?

Reply:
because I use it here: https://github.com/spatialos/UnrealGDK/pull/1771/files/1796da8523603369ec0b0dbd0820c299aa5b56e2..497d13388470843eb97d83e31c2aae923a10039e#diff-5a8d6fc23b23f1dd10137971ead2c993R138
`if (!GeneratedToken.IsEventHandled())`


plus this makes it easier if you want to use these methods somewhere else as you'll be able to check if the event was handled successfully or ont.